scripts/cxg_remaster_v2.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+
+import os
+import sys
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from backend.corpora.dataset_processing.remaster_cxg_v2 import compute
+
+"""
+Re-master an existing "X" tiledb array in a CXG
+Creates two new arrays (Xr and Xc) with the same data but row- and column-optimized schemas
```

Comment:
"X and Xc", right? Tho `Xr and Xc` seems better.

Reply:
I forgot to update this docstring. I changed it to "X" and "Xc" because it makes backwards compatibility in explorer less verbose.